internal/run/templates/templates.go

```diff
@@ -29,10 +30,11 @@ type Templates struct {
 func Parse(renderTermColors RenderTermColorsType) *Templates {
 	templateFunctions := template.FuncMap{
 		"rate": func(duration time.Duration, count uint64) uint64 {
-			if uint64(duration/time.Second) == 0 {
+			if count == 0 || duration == 0 {
```

Comment:
nit: I think it's still valid, but I'm not sure you need to return early on `count == 0`. Am I missing something?